ocaml-lsp-server/test/e2e-new/type_enclosing.ml

```diff
@@ -0,0 +1,491 @@
+open Test.Import
+
+module Util = struct
+  let call_type_enclosing ?(verbosity = 0) ?range_end client position index =
+    let uri = DocumentUri.of_path "test.ml" in
+    let text_document = TextDocumentIdentifier.create ~uri in
+    let params =
+      `Assoc
+        ([ ("textDocument", TextDocumentIdentifier.yojson_of_t text_document)
```

Comment:
How about just exporting the type from `req_type_enclosing`? Writing out the json here will just make it harder to sync the test with the actual code.